app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+    <!-- LOCALIZATION NOTE (restart_dialog_button):  The string 'restart_dialog_button' is used to
+         label the button the user presses to restart the application in the restart dialog. -->
     <string name="restart_dialog_button">Restart Now</string>
+    <!-- LOCALIZATION NOTE (developer_options_remote_debugging): The string 'developer_options_remote_debugging'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle remote debugging
+         of web content in the application. -->
     <string name="developer_options_remote_debugging">Enable Remote Debugging</string>
+    <!-- LOCALIZATION NOTE (developer_options_show_console): The string 'developer_options_show_console'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle redirecting
+         console output to the Android logcat. -->
     <string name="developer_options_show_console">Redirect Console to Logcat</string>
+    <!-- LOCALIZATION NOTE (developer_options_env_override): The string 'developer_options_env_override'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle loading externally
+         generated environments and overriding the ones built in. -->
     <string name="developer_options_env_override">Enable Environment Override</string>
+    <!-- LOCALIZATION NOTE (developer_options_multiprocess): The string 'developer_options_multiprocess'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle running content
+         in a separate process. -->
     <string name="developer_options_multiprocess">Enable Multiprocess</string>
+    <!-- LOCALIZATION NOTE (developer_options_ua_mode): The string 'developer_options_ua_mode' is
+         used to label a set of radio buttons that allow the user to change the user agent (UA) string
+         of the browser. -->
     <string name="developer_options_ua_mode">User Agent Mode</string>
+    <!-- LOCALIZATION NOTE (developer_options_events): FIXME This option should be removed -->
     <string name="developer_options_events">Input Events</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_density">Display Density:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_window_size">Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_mouse">Mouse</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_touch">Touch</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_edit">Edit</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_save">Save</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_dpi">Display DPI:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_max_window_size">Max. Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
```

Comment:
we'll want to explain what MSAA stands for and what it means (as well as in the notes for the related strings below)